ts/features/payments/payment/screens/WalletPaymentInputFiscalCodeScreen.tsx

```diff
@@ -86,6 +97,17 @@ const WalletPaymentInputFiscalCodeScreen = () => {
     );
   };
 
+  const handleContinueClick = () =>
+    O.fold(
+      () => {
+        Keyboard.dismiss();
+        focusTextInput();
+      },
+      _some => navigateToTransactionSummary()
+    )(inputState.fiscalCode);
```

Comment:
```suggestion
const handleContinueClick = () =>
    pipe(
      inputState.fiscalCode,
      O.fold(() => {
        Keyboard.dismiss();
        focusTextInput();
      }, navigateToTransactionSummary)
    );
```

Reply:
not a huge fan, but why not? adding it to next commit

Reply:
@forrest57 If you want to go fully functional, you need to be able to read the code top-bottom, left-right. Your solution is bottom-top, left-right. :)